### DISCUSSION TITLE
[WASM] Getting rid of useless 'Microsoft.AspNetCore.Hosting.Diagnostics' messages

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#7751] [General]
My VS output window is spammed with 'Microsoft.AspNetCore.Hosting.Diagnostics' messages. How could I get rid of those?

I tried:
```
global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory = Microsoft.Extensions.Logging.LoggerFactory.Create(builder
=> {
    Microsoft.Extensions.Logging.LoggingBuilderExtensions.ClearProviders(builder);
    Microsoft.Extensions.Logging.LoggingBuilderExtensions.SetMinimumLevel(builder, Microsoft.Extensions.Logging.LogLevel.Critical);
});
```
... as first instruction in APP.ctor()

However:
a) it did not work
b) I had to install (back) 3 (!) NuGet packages for the sole purpose of trying to get rid of log spam.

Is there a working and more efficient way to get of that log spam?

Thanks in advance

**Comment by jeromelaban** [comment]
Thanks. I'm assuming you're talking about messages in the VS debugger window while debugging. Those messages cannot be removed from the app, and there's no configuration for it at this point.

**Reply by DierkDroth** [reply]
Thanks for your feedback @jeromelaban. Your assumption is correct.

Well, more stuff on MS's plate then...
